pkg/daemon/daemon.go

```diff
@@ -621,8 +615,8 @@ func (dn *Daemon) annotateNode(node, value string) error {
 func (dn *Daemon) drainNode(name string) {
 	glog.Info("drainNode(): Update prepared")
 	var err error
-
-	err = wait.PollImmediateUntil(3*time.Second, func() (bool, error) {
+	// wait a random time to avoid all the nodes drain at the same time
+	err = wait.PollUntil(time.Duration(rand.Intn(15)+1)*time.Second, func() (bool, error) {
```

Comment:
If an error occurs when waiting for the pool, shall it return the err instead of continuing drain process below (annotate node etc)?

Reply:
This poll is just waiting for the `drainalbe` to become `true`, so I don't think any error would happen here. The problem is that it will wait forever and block the configure daemon from handling any new requests.
However, in a large cluster, the waiting time could be very long. So I am not sure whether we should put a timeout here.